site/pt-br/index.html

```diff
@@ -0,0 +1,680 @@
+    <!--[if lt IE 9]>
+        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+        ga('create', 'UA-71645852-1', 'auto');
+        ga('send', 'pageview');
+
+    </script>
+
+</head>
+
+<body>
+
+    <!-- Navigation -->
+    <nav class="navbar navbar-default navbar-fixed-top topnav" role="navigation">
+        <div class="container topnav">
+            <!-- Brand and toggle get grouped for better mobile display -->
+            <div class="navbar-header">
+                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <img src="../img/eclipse-collections-logo.png" style="width:175px;height:50px;" align="left" alt="Eclipse Collections">
+            </div>
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav navbar-right">
+                    <li>
+                        <a href="#start">Comece já</a>
+                    </li>
+                    <li>
+                        <a href="#concept">Conceito</a>
+                    </li>
+                    <li>
+                        <a href="#learn">Aprenda</a>
+                    </li>
+                    <li>
+                        <a href="#history">Histórico</a>
+                    </li>
+                    <li>
+                        <a href="#refGuide">Guia de referência</a>
+                    </li>
+                    <li>
+                        <a href="#sourceCode">Código Fonte</a>
+                    </li>
+                    <li>
+                        <a href="#contribute">Contribua</a>
+                    </li>
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                           aria-expanded="false">Javadoc <span class="caret"></span></a>
+                        <ul class="dropdown-menu">
+                            <li><a href="javadoc/9.0.0">9.0.0</a></li>
```

Comment:
../javadoc/9.0.0
and similarly for others
Likewise can you please fix it in Russian version too. Thanks!